packages/core/src/application.ts

```diff
@@ -234,16 +237,28 @@ export class Application extends Context {
   ) {
     options = options || {version: '2.0', format: 'json'};
     let specObj = this.getApiSpec();
+    let appPort = await this.get('http.port');
+    // TODO make this more robust i.e. get host, protocol, etc. dynamically
+    let appUrl = 'http://127.0.0.1:' + appPort + '/swagger';
     if (options.version === '3.0.0') {
       specObj = await swagger2openapi.convertObj(specObj, {direct: true});
+      appUrl.replace('swagger', 'openapi');
```

Comment:
I don't think `replace` will change `appUrl` at all.

Reply:
Good catch. I will fix it pending on discussion of https://github.com/strongloop/loopback-next/pull/570#pullrequestreview-62490788.